pipeline/transformer/get_dlq_failures.py

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env python3
+"""
+This is a crude script for analysing errors in the transformer DLQs.
+
+It:
+
+*   saves a list of all messages from the DLQ
+*   queries the logging cluster for errors with the same ID
```

Comment:
This looks like a really handy tool.

Reply:
I think it's potentially extensible to other pipeline apps also, if we log the IDs correctly.